kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ReplaceQuestions.vue

```diff
@@ -1,21 +1,415 @@
+            answer: 'bit',
+            selected: false,
+          },
+          {
+            index: 'B',
+            answer: 'bat',
+            selected: false,
+          },
+          {
+            index: 'C',
+            answer: 'but',
+            selected: false,
+          },
+          {
+            index: 'D',
+            answer: 'bite',
+            selected: false,
+          },
+          {
+            index: 'E',
+            answer: 'bet',
+            selected: false,
+          },
+          {
+            index: 'F',
+            answer: 'bait',
+            selected: true,
+          },
+        ],
+        placeholderList: [
+          'Letters and Sounds Practice',
+          'Mathematics and Geometry',
+          'Algebra Practice',
+          'Long and Short Vowel Patterns: VCV and VCC Practice',
+          'Letters and Sounds Practice',
+          'Mathematics and Geometry',
+          'Algebra Practice',
+          'Long and Short Vowel Patterns: VCV and VCC Practice',
+        ],
+      };
+    },
+    $trs: {
+      replaceQuestionText: {
+        message: 'Replace questions',
+        context: 'Title for the replace questions on the side panel',
+      },
+      replaceQuestionDescription: {
+        message: 'Replace with questions from previously selected folders',
+        context: 'Description for the replace questions title',
+      },
+      shortNote: {
+        message: 'Short <e>, [e]</e>',
+        context: 'Short description about the section question.',
+      },
+      chooseAnswer: {
+        message: 'Choose 1 answer:',
+        context: 'Directs the user to select answer from the list of available options',
+      },
+      selectAll: {
+        message: 'Select all',
+        context: 'Option for use to select all questions at once',
+      },
+      questionTitle: {
+        message: 'Select the word that has the following vowel sound.',
+        context: 'Question title in a particular section.',
+      },
+      sectionTitle: {
+        message: 'Section title',
+        context: 'Title for the section on the replace question side panel.',
+      },
+    },
   };
 
 </script>
+
+
+<style scoped>
+  .remove-button-style {
+    width: 100%;
+    padding: 0;
+    background-color: transparent;
+    border: 0;
+  }
+  .flex-div {
+    display: flex;
+  }
+  .left-column-alignment-style {
+    display: inline-flex;
+  }
+  .check-box-style {
+    margin-top: 0.5em;
+    margin-left: 0.5em;
+  }
+  .occupy-remaining-space {
+    flex-grow: 1;
+  }
+  .limit-height {
+    margin-top: 0.5em;
+    margin-bottom: 0.5em;
+  }
+  .right-alignment-style {
+    float: right;
+    margin-top: 1em;
+  }
+
+  .horizontal-border{
+    color:#DEDEDE;
+  }
+
+  .text-align-start {
+    text-align: start;
+  }
+  .text-vertical-spacing {
+    margin-top: 0.5em;
+  }
+  .float-item-left-style {
+    float: right;
+    margin-top: 1em;
+  }
+  .choose-answer-style{
+    background-color:#FAFAFA;
```

Comment:
We don't hardcode colors into Kolibri (except a very few cases due to technical limitations). Tokens are used instead. This applies to other places with colors in this PR. Please see https://design-system.learningequality.org/colors/